src/VirtoCommerce.Platform.Web/Controllers/HomeController.cs

```diff
@@ -31,6 +31,7 @@ public ActionResult Index()
             var model = new IndexModel
             {
                 PlatformVersion = Core.Common.PlatformVersion.CurrentVersion.ToString(),
+                ProductVersion = Core.Common.PlatformVersion.ProductVersion.Substring(0, 20),
```

Comment:
What does this magic number mean?

Reply:
![image](https://user-images.githubusercontent.com/8361000/67187098-c9cb8a80-f3e9-11e9-8887-d43e84145d1b.png)
refactored this code.